pkgs/applications/office/wpsoffice/default.nix

```diff
@@ -77,18 +62,31 @@ stdenv.mkDerivation rec{
     cp -r usr/* $out
     # Avoid forbidden reference error due use of patchelf
     rm -r *
-    for i in wps wpp et wpspdf; do
-      patchelf \
-        --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
-        --force-rpath --set-rpath "$(patchelf --print-rpath $prefix/office6/$i):${stdenv.cc.cc.lib}/lib64:${libPath}" \
-        $prefix/office6/$i
+    bins=$(find $prefix/office6 -maxdepth 1 -executable ! -type d|grep -v '.*\.so\.*')
```

Comment:
Looks like you are reinventing `autoPatchelfHook` here https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/auto-patchelf.sh